buddyboss/buddyboss-platform
# #2858 - fixed topics same slug tag does not add issue

**Related platform api PR**
https://github.com/buddyboss/buddyboss-platform-api/pull/329

**Related app native issue for QA**
buddyboss/buddyboss-app-native#1598

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2858  .

### How to test the changes in this Pull Request:

1. Create a discussion topic with tag like "@c#"
2. Edit the topic and add new tag like "@c++"
3. Save it and check the tags 
4. See that "@c++" has been added now

please check for **create topic, edit topic, create reply, edit reply**

### Proof Screenshots or Video
https://www.loom.com/share/49b75a50486640cc826330211040c7b2



### Other information:
